ina260/ina260.go

```diff
@@ -0,0 +1,118 @@
+package ina260
+
+import "tinygo.org/x/drivers"
+
+// Device wraps an I2C connection to an INA260 device.
+type Device struct {
+	bus     drivers.I2C
+	Address uint16
+}
+
+// Config holds the configuration of the INA260 device.
+type Config struct {
+	// One of AVGMODE_XXX
+	AverageMode byte
+
+	// One of CONVTIME_XXXXUSEC
+	VoltConvTime byte
+
+	// One of CONVTIME_XXXXUSEC
+	CurrentConvTime byte
+
+	// Multiple of MODE_XXXX
+	Mode byte
+}
+
+// New creates a new INA260 connection. The I2C bus must already be
+// configured.
+//
+// This function only creates the Device object, it does not touch the device.
+func New(bus drivers.I2C, address uint16) Device {
```

Comment:
I think it is better to define this as `func New(bus drivers.I2C) Device` and then just set `Address` to special value only if needed before using. See https://github.com/tinygo-org/drivers/blob/release/adt7410/adt7410.go#L39-L45

Reply:
That makes sense - pushed an update